### DISCUSSION TITLE
use MMKV instead of AsyncStorage

### DISCUSSION THREAD

**Discussion by ryskin** [infinitered/ignite#2304] [Ideas]
I had a problem in my project with the limitation of AsyncStorage that led to crashes, also it was really slow, but when I moved to MMKV all become really quick and stable.

I used this library https://github.com/ammarahm-ed/react-native-mmkv-storage

**Comment by jamonholmgren** [comment]
While we don't use MMKV for Ignite by default (since Expo doesn't support it), we do have an Ignite Cookbook recipe for it. Check it out:

https://infinitered.github.io/ignite-cookbook/docs/MigratingToMMKV

**Reply by ryskin** [reply]
I see, it makes sense. Thanks
